saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -1921,7 +1921,9 @@ def test_event_delivery_retry(mocked_webhook_send, event_delivery, settings):
 
     # then
     mocked_webhook_send.assert_called_once_with(
-        event_delivery.pk, telemetry_context=ANY
+        kwargs={"event_delivery_id": event_delivery.pk, "telemetry_context": ANY},
+        queue=settings.WEBHOOK_CELERY_QUEUE_NAME,
+        MessageGroupId="mirumee.com:saleor.app.test",
```

Comment:
Maybe let's use something more generic than `mirumee.com`?

Reply:
I fixed that, but it touched a whole lot of tests.